resotoworker/resotoworker/config.py

```diff
@@ -9,6 +9,13 @@ def add_config(config: Config) -> None:
     config.add_config(ResotoWorkerConfig)
 
 
+@define
+class HomeDirectoryFile:
+    kind: ClassVar[str] = "resotoworker_home_directory_file"
```

Comment:
This should be part of resotolib and can be used by anything, not just resotoworker.

Reply:
That's a great idea for the future, when we have a need to use this feature not only in the resotoworker but also in other components. But let's keep it simple for now. We can always move it to resotolib when the need is there.